src/cncf_distribution/mod.rs

```diff
@@ -0,0 +1,116 @@
+use testcontainers::{core::WaitFor, Image};
+
+const NAME: &str = "registry";
+const TAG: &str = "2";
+
+/// Module to work with a custom Docker registry inside of tests.
+///
+/// Starts an instance of [`CNCF Distribution`], an easy to use registry for container images.
+///
+/// # Example
+/// ```
+/// use testcontainers::clients;
+/// use testcontainers_modules::cncf_distribution;
+///
+/// let docker = clients::Cli::default();
+/// let registry = docker.run(cncf_distribution::CncfDistribution);
+///
+/// let image_name = "test";
+/// let image_tag = format!("localhost:{}/{image_name}", registry.get_host_port_ipv4(5000));
+///
+/// // now you can push an image tagged with `image_tag` and pull it afterwards
+/// ```
+///
+/// [`CNCF Distribution`]: https://distribution.github.io/distribution/
+#[derive(Debug, Default, Clone)]
+pub struct CncfDistribution;
+
+impl Image for CncfDistribution {
+    type Args = ();
+
+    fn name(&self) -> String {
+        NAME.to_owned()
+    }
+
+    fn tag(&self) -> String {
+        TAG.to_owned()
+    }
+
+    fn ready_conditions(&self) -> Vec<WaitFor> {
+        vec![WaitFor::millis(500)]
```

Comment:
Is there any way to depend on stdout/stderr? It's much better, time is too flaky condition usually

Reply:
Yes, I have just switched it from `millis` to `message_on_stderr`